packages/elements-dev-portal/src/containers/StoplightProject.tsx

```diff
@@ -126,7 +119,7 @@ const StoplightProjectImpl: React.FC<StoplightProjectProps> = ({
     elem = (
       <NodeContent
         node={node}
-        Link={ReactRouterMarkdownLink!}
+        Link={Link}
```

Comment:
@lottamus I believe the problem was with scrolling to anchor links on other pages. So if you were linking to another article (/anotherarticle#heading2) it would strip out the anchor link part.